### DISCUSSION TITLE
Cloning into 'E:\---'... fatal: unable to access();

### DISCUSSION THREAD

**Discussion by NihaalKnight87** [community/community#21855] [New to GitHub]
<p><strong>Cloning into ‘E:—’… fatal: unable to access ‘<a href="https://github.com/NIhaalknight87/DE_03-ESG.git/" title="https://github.com/NIhaalknight87/DE_03-ESG.git/" rel="nofollow noopener">https://github.com/------.git/</a>’: schannel: next InitializeSecurityContext failed: Unknown error (0x80092013) - The revocation function was unable to check revocation because the<em> revocation server</em> was offline. <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/8c9d3bf080e253b60b7063b8b01fab90a1e43013.png" data-download-href="/uploads/short-url/k3VIxJ09DQQuwCrYfJafvC9h3z5.png?dl=1" title="2018-08-05 (1).png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/30192034/181067139-b004c173-a186-4fc1-9edf-daad73c76d18.png" alt="2018-08-05 (1).png" data-base62-sha1="k3VIxJ09DQQuwCrYfJafvC9h3z5" width="690" height="293" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/8c9d3bf080e253b60b7063b8b01fab90a1e43013_2_690x293.png, https://aws1.discourse-cdn.com/github/original/1X/8c9d3bf080e253b60b7063b8b01fab90a1e43013.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/8c9d3bf080e253b60b7063b8b01fab90a1e43013.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/8c9d3bf080e253b60b7063b8b01fab90a1e43013_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2018-08-05 (1).png</span><span class="informations">999×425 125 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div></strong></p>

**Comment by NihaalKnight87** [comment]
<p>Is it A bug Are How Can I Fix it</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/nihaalknight87">@nihaalknight87</a> this is a known issue that can occur with GitHub Desktop. This error is usually caused by your network blocking the attempt by Windows to check the revocation status of a certificate, which then causes the whole operation to error. You can find a workaround for this issue here:</p>
<p><a href="https://github.com/desktop/desktop/blob/development/docs/known-issues.md#certificate-revocation-check-fails---3326" rel="nofollow noopener">https://github.com/desktop/desktop/blob/development/docs/known-issues.md#certificate-revocation-check-fails—3326</a></p>
<p>If you have any questions about the workaround, please let me know.</p>

**Comment by pavanHCL** [comment]
<p>i did’nt get clerification on this could you please help me out by providing in details </p>

**Comment by ijohnson222** [comment]
<p>This is a recurrent problem on Windows, it would be good if - as at least one person has suggested - the error message was more informative eg. suggesting to look at recently installed software / updates which might be blocking it.</p>
<p>Plus, as a naive user, I have no idea what a revocation server is, why I need it, or why it should be unavailable. Is it an actual server which might be down? From the comments on (many) past reports it is more likely blocking of access.</p>
<p>However, in the past I have got this error then a day or so later it has gone away without me doing anything, which suggests that it is not blocking of access on my machine as most comments suggest but a fault further down the line. Perhaps this should be part of the explanation.</p>
<p>Also, the reference to the workaround goes to a 404.</p>

**Comment by georgiaoc** [comment]
<p>I’ve also been having this issue intermittently for the past two weeks, despite having no problems for the 5 prior months that I have been using it with this repository. As ‘ijohnson222’ said, it does go away after a day or two, but I still find it disconcerting and would like to see if there’s anything I can do about it.</p>
<p>Since the link you found helpful no longer exists – could you please reproduce the solution here?</p>
<p>(also another user recently reported the same issue so it seems it has not really been resolved: <a href="https://github.community/t5/forums/replypage/board-id/Board_1/message-id/8642" rel="nofollow noopener">https://github.community/t5/forums/replypage/board-id/Board_1/message-id/8642) </a> )</p>

**Comment by georgiaoc** [comment]
<p>Hi Steve,</p>
<p>Could you please review new messages added to this tread since your solution was posted? The link with the solution no longer works. </p>
<p>Thank you in advance! </p>
<p>Georgia </p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/georgiaoc">@georgiaoc</a>,</p>
<p>Thank you for being here and for reaching out!  Can you try this link to see if this is the information that you are looking for: <a href="https://github.com/desktop/desktop/blob/development/docs/known-issues.md#certificate-revocation-check-fails---3326" rel="nofollow noopener">https://github.com/desktop/desktop/blob/development/docs/known-issues.md#certificate-revocation-check-fails—3326</a> ?</p>

**Comment by georgiaoc** [comment]
<p>Great, thank you! </p>
